__tests__/app.test.js

```diff
@@ -185,51 +185,73 @@ describe("patch article votes", () => {
-    });
-    test('400: should return bad reqyest', () => {
-        return request(app)
-        .patch("/api/articles/banana")
-        .send({ inc_votes: 1 })
-        .expect(400)
-        .then(({ body }) => {
-            expect(body.msg).toBe(`ERROR: bad request. ensure you use a valid article ID number`);
-        });
-    });
-})
+      });
+  });
+  test("404: should return not found", () => {
+    return request(app)
+      .patch("/api/articles/99")
+      .send({ inc_votes: 1 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe(`ERROR: no article with that id found`);
+      });
+  });
+  test("400: should return bad reqyest", () => {
+    return request(app)
+      .patch("/api/articles/banana")
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          `ERROR: bad request. ensure you use a valid article ID number`
+        );
+      });
+  });
+});
 
 describe("delete article comments", () => {
-    test("204: should delete a comment", () => {
-        return request(app)
-       .delete("/api/comments/1")
-       .expect(204)
-       .then(({ body }) => {
-            expect(body).toEqual({});
-        })
-    })
-    test("404: should return not found", () => {
-        return request(app)
-       .delete("/api/comments/999")
-       .expect(404)
-       .then(({ body }) => {
-            expect(body.msg).toBe(`ERROR: no article with that id found`);
-        })
-    })
-    test("400: should return bad request", () => {
-        return request(app)
-       .delete("/api/comments/banana")
-       .expect(400)
-       .then(({ body }) => {
-            expect(body.msg).toBe(`ERROR: bad request. ensure you use a valid article ID number`);
-        })
-    })
-})
\ No newline at end of file
+  test("204: should delete a comment", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({});
+      });
+  });
+  test("404: should return not found", () => {
+    return request(app)
+      .delete("/api/comments/999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe(`ERROR: no article with that id found`);
+      });
+  });
+  test("400: should return bad request", () => {
+    return request(app)
+      .delete("/api/comments/banana")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe(
+          `ERROR: bad request. ensure you use a valid article ID number`
+        );
+      });
+  });
+});
+
+describe(" get users", () => {
+  test("200: should get all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            avatar_url: expect.any(String),
+            name: expect.any(String),
+          });

```

Comment:
Include a length check here, how many users are you expecting from the test data?